draft-irtf-cfrg-frost.md

```diff
@@ -229,21 +229,21 @@ following operation.
   Inputs:
   - msg, message to be signed, an octet string
   - SK, private key, a scalar
-  - PK, public key, a group element
 
   Outputs: signature (R, z), a pair of scalar values
 
-  def schnorr_signature_generate(msg, SK, PK):
-    r = G.RandomScalar()
-    R = G.ScalarBaseMult(r)
+  def schnorr_signature_generate(msg, SK):
+    PK = G.ScalarBaseMult(s)
```

Comment:
`s` is not defined

Reply:
right, was a copy&paste from POC code, fixing it right now